bedrock/base/templates/includes/protocol/footer/footer.html

```diff
@@ -71,7 +71,7 @@ <h5 class="mzp-c-footer-heading">
         <section class="mzp-c-footer-section">
           <h5 class="mzp-c-footer-heading-social">{{ ftl('footer-follow-mozilla') }}</h5>
           <ul class="mzp-c-footer-links-social">
-            <li><a class="twitter" href="{{ mozilla_twitter_url() }}" data-link-type="footer" data-link-text="Twitter (@mozilla)">{{ ftl('footer-twitter') }}<span> (@mozilla)</span></a></li>
+            <li><a class="twitter" href="{{ mozilla_twitter_url() }}" data-link-type="footer" data-link-text="Twitter (@mozilla)" aria-label="{{ ftl('footer-x-formerly-twitter') }}">{{ ftl('footer-x', fallback='footer-twitter') }}<span> (@mozilla)</span></a></li>
```

Comment:
I'm not sure we need to add `aria-label` here since the link already contains screen-reader friendly text? Could we maybe simplify it using `<a>X (formerly twitter) @mozilla</a>`?

Reply:
Yeah i think you might be right. I just wanted to make sure it was friendly enough for folks using screenreaders to know what X is (since "x" can sometimes allude to "close" "cancel" "quit" etc)